characters_counter/characters_counter.py

```diff
@@ -0,0 +1,15 @@
+from collections import Counter
+from functools import cache
+
+
+@cache
+def count_characters_occurrence(string: str) -> int:
+    """
+    The function returns the number of characters in the string occuring only
+    once.
+    """
+    characters_occurrence = Counter(string)
+    single_occurrence = sum(
+        [1 for v in characters_occurrence.values() if v == 1]
```

Comment:
What does 1 mean?
1. one as an occurance, this is our Constant within business requirement. Please define constant variable
2. 1 for v in ... I'm not sure what this one is. Try using filter() with condition https://stackoverflow.com/questions/55133338/filter-a-list-of-tuples-based-on-condition

Reply:
Ok, I made changes